.gitignore

```diff
@@ -3,3 +3,4 @@
 
 # VSCode Workspace
 workspace.code-workspace
+.idea
```

Comment:
We aren't accepting any more editor-specific ignore directives. There is a much better approach which involves using a global `gitignore` file: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/